src/nvim/os/pty_process_win.c

```diff
@@ -353,6 +354,7 @@ static void quote_cmd_arg(char *dest, size_t dest_remaining, const char *src)
   //   output: "hello\\\\\"world"
   //   input : hello world\
   //   output: "hello world\\"
+*/
```

Comment:
What is that for? These lines are already commented.

Reply:
```
//   input : hello world\
```
will cause warning with mingw64-gcc 7.2.0, maybe we need other solution

Reply:
Would suggest to just single-quote everything, something like `:341,355s/: \zs.*/\=string(submatch(0))`.

Reply:
Done.